src/styles/blocks/page.scss

```diff
@@ -0,0 +1,11 @@
+.page {
+  &__content {
+    position: relative;
+    height: 100vh;
+    margin: 0;
```

Comment:
You reset body margin, move it to reset.scss

Reply:
I guess there isn't any difference whether I do it in a reset SCSS or page block